assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -39,6 +39,11 @@ describe( 'modules/analytics accounts', () => {
 	let registry;
 	let store;
 
+	const accountName = fixtures.createAccount.account.name;
+	const propertyName = fixtures.createAccount.webproperty.name;
+	const profileName = fixtures.createAccount.profile.name;
+	const timezone = fixtures.createAccount.profile.timezone;
+
```

Comment:
These should be declared more locally to the tests that use them (perhaps within `describe( 'createAccount'`) as these variables aren't relevant to the whole file.

Reply:
👍

Reply:
Moved in https://github.com/google/site-kit-wp/pull/1390/commits/0793a77d37d938987a6496259573efa95ff6152d